internal/cmd/login.go

```diff
@@ -82,6 +83,8 @@ $ infra login --key 1M4CWy9wF5.fAKeKEy5sMLH9ZZzAur0ZIjy`,
 	cmd.Flags().StringVar(&options.AccessKey, "key", "", "Login with an access key")
 	cmd.Flags().StringVar(&options.Provider, "provider", "", "Login with an identity provider")
 	cmd.Flags().BoolVar(&options.SkipTLSVerify, "skip-tls-verify", false, "Skip verifying server TLS certificates")
+	cmd.Flags().Var((*types.StringOrFile)(&options.TrustedCertificate), "tls-trusted-cert", "TLS certificate or CA used by the server")
+	cmd.Flags().Var((*types.StringOrFile)(&options.TrustedFingerprint), "tls-trusted-fingerprint", "SHA256 fingerprint of the server TLS certificate")
```

Comment:
Should the fingerprint just be a string only, rather than a StringOrFile?

Reply:
Could be, I'm not sure if people ever store a fingerprint in a file or if they expect to set it from command line flag